s2p/triangulation.py

```diff
@@ -219,15 +240,13 @@ def height_map(x, y, w, h, rpc1, rpc2, H1, H2, disp, mask, utm_zone, A=None):
         H1, H2 (arrays): 3x3 numpy arrays defining the rectifying homographies
         disp, mask (array): 2D arrays of shape (h, w) representing the diparity
             and mask maps
-        utm_zone (int): desired UTM zone number (between 1 and 60) for the
-            output xyz map
         A (array): 3x3 array with the pointing correction matrix for im2
 
     Returns:
         array of shape (h, w) with the height map
     """
-    xyz, err = disp_to_xyz(rpc1, rpc2, H1, H2, disp, mask, utm_zone, A=A)
-    height_map = xyz[:, :, 2].squeeze()
+    lonlatalt, err = disp_to_xyz(rpc1, rpc2, H1, H2, disp, mask, A=A)
+    height_map = lonlatalt[:, :, 2].squeeze()
```

Comment:
I guess you forgot to rename `lonlatalt` to `xyz` after a recent change you made where `disp_to_lonlatalt` was re-renamed to `disp_to_xyz`.

```suggestion
    xyz, err = disp_to_xyz(rpc1, rpc2, H1, H2, disp, mask, A=A)
    height_map = xyz[:, :, 2].squeeze()
```